daemon/cmd/daemon_main.go

```diff
@@ -1733,9 +1702,8 @@ func initKubeProxyReplacementOptions() {
 				option.Config.EnableExternalIPs = false
 			}
 		} else {
-			log.WithField(logfields.Interface, device).
-				Info("Using auto-derived device for BPF node port")
-			option.Config.Devices = []string{device}
+			log.WithField(logfields.Interface, option.Config.Devices).
+				Info("Using auto-derived devices for BPF node port")
```

Comment:
Might be nice to get the equivalent message for `DirectRoutingDevice` here as well.